README.md

```diff
@@ -730,6 +744,11 @@ An order for placing the rule in `pg_hba.conf`. Defaults to `150`.
 ####`target`
 This provides the target for the rule, and is generally an internal only property. Use with caution.
 
+####`postgresql_version`
+Defaults to the version set in `postgresql::server`.  Use this if you want to manage `pg_hba.conf` without managing the entire postgreSQL instance.
+
+####`manage_pg_hba_conf`
```

Comment:
@DavidS I don't quite track what you are saying.  Are you saying remove this parameter altogether because the only way you'd use it is to set it to true and then change the if statement to be something like....

```
if defined ( 'postgresql::server' ) {
  if $postgresql::server::manage_pg_hba_conf == false {
       fail('postgresql::server::manage_pg_hba_conf has been disabled, so this resource is now unused and redundant, either enable that option or remove this resource from your manifests')
   } else {
     validate_re($type, '^(local|host|hostssl|hostnossl)$',
     "The type you specified [${type}] must be one of: local, host, hostssl, hostnosssl")
}
```

It hadn't occurred to be to check if `postgresql::server` was already defined and I think if that works the way I think it does then I'm happy to make that change as it's much cleaner.